qemu/tests/cfg/virtio_serial_file_transfer.cfg

```diff
@@ -28,6 +28,15 @@
                     file_sender = host
                 - from_guest_to_host:
                     file_sender = guest
+        - boot_with_multiple_serials:
+            variants:
+                - 1_N:
+                    only unix_socket
+                    type = boot_1_N
```

Comment:
Script name at least include feature or device name, thanks.

Reply:
> Script name at least include feature or device name, thanks.

Thanks and update.